scripts/on-prem-archive.sh

```diff
@@ -406,9 +419,9 @@ case "${1:-}" in
 
         if download_hart_if_missing "$local_file" "$slash_ident" "[$pkg_count/$pkg_total]" "$target"; then
           echo "[$pkg_count/$pkg_total] Uploading ${slash_ident} to local depot"
-          checksum=$(b2sum -s=32 ${local_file} | cut -d ' ' -f 4-)
-          curl -so /dev/null -H "Accept: application/json" -H "Content-Type: application/json" -H "Authorization: Bearer $HAB_AUTH_TOKEN" --data-binary "@$local_file" ${depot_url}/v1/depot/pkgs/$slash_ident\?checksum=$checksum
-          hab pkg promote --url ${depot_url} ${slash_ident} stable ${target} || true
```

Comment:
Did the promote get inadvertently deleted here?

Reply:
Crud! Yep, I'll get that added back.